blackbox/docs/sql/information_schema.txt

```diff
@@ -27,41 +27,41 @@ settings like the ``number of shards`` or ``number of replicas``.
+| ``number_of_shards``             | The number of shards the table is currently distributed across                     | ``Integer`` |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``partitioned_by``               | The column used to partition the table                                             | ``String``  |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``reference_generation``         | Not implemented                                                                    |             |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``routing_hash_function``        | The hash function used for internal routing                                        | ``String``  |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``self_referencing_column_name`` | Not implemented                                                                    |             |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``settings``                     | :ref:`conf_table_settings`                                                         | ``Object``  |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``table_catalog``                | The name of the database (always ``CrateDB``)                                      | ``String``  |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``table_name``                   | The name of the table                                                              | ``String``  |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``table_schema``                 | The name of the schema the table belongs to                                        | ``String``  |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``table_type``                   | The type of the table (always ``BASE TABLE``)                                      | ``String``  |
++----------------------------------+------------------------------------------------------------------------------------+-------------+
+| ``version``                      | Information about the CrateDB version the table was created and upgraded under     | ``Object``  |
```

Comment:
aha, how about:

"Two CrateDB versions, corresponding to the version used to create and last upgrade the table"